### PR TITLE
Ensure valid AJAX targets to avoid about:blank errors

### PR DESCRIPTION
**Pull request by RealTreasury**
## Summary
- Validate and sanitize AJAX URLs across admin and public scripts
- Fallback to WordPress `ajaxurl` when plugin URLs are missing or invalid
- Guard against empty URLs causing `about:blank` requests

## Testing
- `find . -name "*.php" -not -path "./vendor/*" -print0 | xargs -0 -n1 php -l`
- `bash tests/run-tests.sh` *(fails: Run `composer install` to install PHPUnit)*

------
https://chatgpt.com/codex/tasks/task_e_68b5da4ee24083318b965b32f840af24

### CODE REVIEW

File: public/js/rtbcb-wizard.js [hunk 1]
```diff
@@ -21,6 +21,13 @@ function isValidUrl(url) {
     }
 }
 
+// Normalize global AJAX URL if provided.
+if ( typeof rtbcb_ajax !== 'undefined' && ! isValidUrl( rtbcb_ajax.ajax_url ) ) {
+    if ( typeof ajaxurl !== 'undefined' && isValidUrl( ajaxurl ) ) {
+        rtbcb_ajax.ajax_url = ajaxurl;
+    }
```
[the remark this answers is not in the document]
Comment (RealTreasury) [reply public/js/rtbcb-wizard.js:28]: > **[P1] Sync minified assets with updated URL validation**
> 
> The new AJAX URL normalization logic is added only to the unminified source. In production WordPress sites `SCRIPT_DEBUG` defaults to false, so the enqueued asset is `rtbcb-wizard.min.js`. That file was not regenerated in this commit and still contains the pre‑existing code, so the about:blank fix introduced here will never execute for most users. Please rebuild or commit the updated minified bundle alongside the source changes.
> 
> Useful? React with 👍 / 👎.

@Codex please fix
